lib/spack/spack/test/main.py

```diff
@@ -4,12 +4,19 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 import os
+import sys
+
+import pytest
 
 import llnl.util.filesystem as fs
 
 import spack.paths
 from spack.main import get_version, main
 
+pytestmark = pytest.mark.skipif(
+    sys.platform == 'win32',
+    reason="Test functionality support but failing on Win")
```

Comment:
Apologies but I'm having trouble parsing this, e.g.

> Functionality is supported but tests are failing on windows

would be more clear (or if I got that wrong, then however as long as there is some elaboration).